### PR TITLE
Adds E-N (borgi)

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

Adds E-N, the borgi that sparks.

image

Mappers feel free to add him to a spawn or something.

Also minor, adds a sprite for the generic (non-corgi) dog

image

#### Changelog

:cl:  
rscadd: Adds E-N, the borgi
rscadd: Adds a sprite for generic dog
/:cl:

### CODE REVIEW

**Review by Partheo — APPROVED** [review]

File: code/modules/mob/living/simple_animal/friendly/dog.dm [hunk 1]
```diff
@@ -52,6 +57,44 @@
...
 	return ..()
 
+/mob/living/simple_animal/pet/dog/corgi/borgi
+	name = "E-N"
+	real_name = "E-N"
+	desc = "It's a borgi."
+	icon_state = "borgi"
+	icon_living = "borgi"
+	icon_dead = "borgi_dead"
+	var/emagged = 0
```
Comment (jcatjmeow) [inline code/modules/mob/living/simple_animal/friendly/dog.dm]:
```suggestion
	var/emagged = FALSE
```

File: code/modules/mob/living/simple_animal/friendly/dog.dm [hunk 2]
```diff
@@ -52,6 +57,44 @@
...
+	minbodytemp = 0
+	loot = list(/obj/effect/decal/cleanable/robot_debris)
+	del_on_death = 1
```
Comment (jcatjmeow) [inline code/modules/mob/living/simple_animal/friendly/dog.dm]:
```suggestion
	del_on_death = TRUE
```

File: code/modules/mob/living/simple_animal/friendly/dog.dm [hunk 3]
```diff
@@ -52,6 +57,44 @@
...
+	deathmessage = "blows apart!"
+	animal_species = /mob/living/simple_animal/pet/dog/corgi/borgi
+	nofur = TRUE
+	
+/mob/living/simple_animal/pet/dog/corgi/borgi/emag_act(user as mob)
+	if(!emagged)
+		emagged = 1
```
Comment (jcatjmeow) [inline code/modules/mob/living/simple_animal/friendly/dog.dm]:
```suggestion
		emagged = TRUE
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/friendly/dog.dm (hunk 3 above)
Comment (swissloaf) [reply code/modules/mob/living/simple_animal/friendly/dog.dm]: does not matter but ok

File: code/modules/mob/living/simple_animal/friendly/dog.dm (hunk 2 above)
Comment (swissloaf) [reply code/modules/mob/living/simple_animal/friendly/dog.dm]: does not matter but ok

**Comment by swissloaf** [comment]
cucked ktl

**Pull request merged**